tutorial.md

```diff
@@ -307,13 +307,21 @@ First we need to setup a Pod sandbox using a Pod configuration, which can be fou
 cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
 ```
 
+
+Copy the policy template into /etc/containers/:
+
+```
+sudo mkdir /etc/containers/
+sudo cp test/policy.json /etc/containers
```

Comment:
This is also available in skopeo-containers.

Reply:
It probably gets copied over when installing from .deb, but when you build skopeo from source you find that missing. Does it make sense to leave that "check" in the tutorial? (Ubuntu 18.04 is out and the skopeo repo hasn't been updated yet, it seems).

Reply:
Maybe add a comment that if the file does not exist do the copy.

Reply:
Makes sense. Thanks!